assets/prefabs/Equipment/Silver/SilverPen.prefab

```diff
@@ -0,0 +1,20 @@
+{
+    "parent" : "engine:iconItem",
+    "Item" : {
+        "icon" : "Equipment:SilverPen"
+    },
+       "DisplayName" : {
+        "name" : "Silver Pen",
+        "description" : "A pen that can be transformed into a sword and can always be found in the user's pocket at any given time. Best for defence as well as for attack"
+    },
+    "EquipmentItem" : {
+        "level" : 9,
+        "quality" : 18,
+        "type" : "Pen/Sword",
```

Comment:
The pen is indeed mightier than the sword! :D However I don't think we have a type (or location) named this. To be able to equip something on the character screen it needs to have a type/location that exists. In this case probably:

```
    "type" : "Melee Weapon",
    "location" : "Weapon",
```